test/e2e/pvc_label_selector.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
I need more comment for this test.. from the looks of it this is testing basic feature of helping claims map to volumes which might be a good general test within k8s but not sure if I am missing some vSphere specific test here.